### DISCUSSION TITLE
Focus Examples on Website broken? Clicking buttons does nothing.

### DISCUSSION THREAD

**Discussion by christoferd** [alpinejs/alpine#3356] [4. General]
Please delete this discussion. It works now. 25-Dec-2022

Are the website examples broken on the Focus page?

[https://alpinejs.dev/plugins/focus](https://alpinejs.dev/plugins/focus)

I'm on Windows 11
Brave Browser latest version
All plugins disabled
No errors in DevTools > Console. No issues found in DevTools > Network.

None of the examples work for me.

**Comment by ekwoka** [comment]
All works fine on Samsung Browser.
